localstack/services/awslambda/provider.py

```diff
@@ -154,11 +154,11 @@
 )
 from localstack.services.awslambda.invocation.lambda_service import (
     LambdaService,
-    lambda_stores,
     store_lambda_archive,
     store_s3_bucket_archive,
 )
 from localstack.services.awslambda.invocation.models import LambdaStore
+from localstack.services.awslambda.lambda_utils import get_lambda_store
```

Comment:
Please do not mix or in any way entangle the old and the new provider.
Would be nice if the new one (already using stores, in invocation/models.py) could continue using it, and only the old one would be changed.

Reply:
Yikes, didn't realise there is already a `LambdaStore` at `.invocation.models`